petab/lint.py

```diff
@@ -621,6 +625,21 @@ def assert_noise_distributions_valid(observable_df: pd.DataFrame) -> None:
                     f"table: {distr}.")
 
 
+def assert_observable_id_is_unique(observable_df: pd.DataFrame) -> None:
```

Comment:
```suggestion
def assert_unique_observable_ids(observable_df: pd.DataFrame) -> None:
```

Reply:
Name is indeed better. But since I essentially copied `assert_parameter_id_is_unique`, maybe if I change the name for the new function I should change it for the old too.

Reply:
right. yes, makes sense to change the old one too then.